container-chains/nodes/frontier/src/service.rs

```diff
@@ -18,7 +18,6 @@
 
 #[allow(deprecated)]
 use {
-    crate::client::TemplateRuntimeExecutor,
```

Comment:
We need to remove this native executor as well right?

Reply:
yep

Reply:
Updated